### PR TITLE
Adds hackerYou logo

### PR DESCRIPTION
**Pull request by Rchristiani**

### CODE REVIEW

File: images/users/hackeryou.svg [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        </g>
+    </g>
+</svg>
```
Comment (thejameskyle) [inline images/users/hackeryou.svg]: Can you run this file through `svgo`?

Comment (thejameskyle) [inline images/users/hackeryou.svg]: Also, the dimensions need to be `500x200 (2.5x1)`

**Comment by Rchristiani** [comment]
I was just in the middle of doing that! I think this fixes those issues.

**Comment by thejameskyle** [comment]
Thanks http://babeljs.io/users/#hackerYou

**Pull request merged**